### ISSUE TITLE
Make the CLI more fault tolerant

### ISSUE THREAD

**Issue by pawelblaszczyk5** [tokenami/tokenami#151]
Per our Discord discussion for later investigation 😃

Currently, the CLI doesn't handle gracefully any errors - e.g. it crashes when it encounters syntax error. It's especially painful when using something like auto-saving in IDE, editing config can force you to restart the tokenami CLI. Maybe the CLI could give some warning so user can act on unexpected errors.

**Comment by nialldbarber** [comment]
@jjenzz @pawelblaszczyk5 I found the question `Which file paths should Tokenami watch for its properties?` which eventually gets added to `include: ["./src/**/*.{ts,tsx}"]` in `.tokenami/tokenami.config.ts` confusing, as I mistyped my file path (I was using `src`) and didn't realise why my files were not being scanned for tokenami properties. A small issue, but lowering the barrier to using the library might be useful. 

With that in mind, it might be nice to have the option to either, choose between `src` and `app` as the root directory, or, to have the option to write your own root directory and assume the rest (`/**/*.{ts,tsx}`)? 

And also once you've completed setup, it would be cool to say what happened, i.e. (config successfully created at `./tokenami/`)? 

I'd be happy to create a PR if you think these would be beneficial changes?
